text/0029-delegation-cap.md

```diff
@@ -0,0 +1,67 @@
+# Proportional cap on delegation
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0029](https://github.com/casperlabs/ceps/pull/0029)
+
+We propose introduction of a delegation cap, or a proportional limit on delegation relative to own staked tokens.
+
+## Motivation
+
+[motivation]: #motivation
+
+We expect all users of the platform to assess relevant risks of interacting with various features that we supply, as both the performance of the platform and the outcomes of any interaction are uncertain in a multi-user, concurrent environment. Consequently, we would normally expect that users avoid delegating to validators with a highly skewed ratio of delegated to own tokens, because such validators would sustain only weak penalties for equivocation relative to their weight. However, in order to enhance platform security, we propose to limit overdelegation using a soft cap that does not compromise platform UX and favors delegators over validators.
+
+## Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+We propose to limit the amount of delegated tokens contributing to a bid to a multiplier X of validator’s own token. For example, if a prospective validator’s standing bid is 100 tokens and X is set to 1.2, up to 20 tokens may be delegated. Should a bid be decreased to the point where the quantity of existing delegated tokens violates this constraint, delegated tokens above the limits, in aggregate, should no longer “count” for auction result determination or reward weight. 
```

Comment:
There is only downside to _validators_ here. There are two cases, either the limit is respected (in which case all tokens behave normally), or, else, the limit is breached, in which case delegator tokens are, in a very weak sense, more valuable than validator tokens (since the delegators start eating into validator rewards).